USER_MANUAL.md

```diff
@@ -628,7 +673,526 @@ The measured total site level emission rate on January 30th is assumed to have l
+
+### &lt;reporting_delay&gt;
+
+**Data type:** Numeric (Integer)
+
+**Default input:** 2
+
+**Description:** The number of days that pass between the end of a survey (when a site is flagged or emissions are tagged) and when the duty holder is informed. The reporting delay is then followed by the [repair delay](#delay).
+
+**Notes on acquisition:** Get this information from the service provider.
+
+**Notes of caution:** Many service providers have automated systems for reporting emissions as soon as they are found and tagged. However, some companies still provide paper or pdf reports days or even weeks later. It is important to understand the expectations between the duty holder and the service provider.
+
+### &lt;time_between_sites&gt;
+
+**Description:** The time between sites can be specified as a list in the parameter file or provided in a CSV file format.
+
+#### file (time_between_sites)
+
+**Data type:** String
+
+**Default input:** None
+
+**Description:** A string denoting the filename of a csv file containing travel times. The file should include one row, with a column header in row 1 of `time_between_sites`
+
+**Notes on acquisition:** Each value should represent not only driving time, but all time spent not conducting surveys (driving, breaks, meals, break downs, trains, etc.) This data should be scraped from historical GPS data associated with LDAR survey crews, ideally for the facilities under evaluation.
+
+**Notes of caution:** These data may be difficult to acquire and may lack representativeness.
+
+#### values (time_between_sites)
+
+**Data type:** List of Integers
+
+**Default input:** [30]
+
+**Description:** The list of numbers denotes the time in minutes required to plan, travel, setup, take down, required in between surveys. A value is selected at random from the provided list.
+
+### &lt;scheduling&gt;
+
+#### &lt;deployment_months&gt;
+
+**Data type:** List of integers
+
+**Default input:** [1,2,3,4,5,6,7,8,9,10,11,12]
+
+**Description:** A list of months used for scheduling. Methods can only be deployed during these months. For example, [8,9] indicates methods can only be deployed in August and September. If not defined, LDAR-Sim assumes methods can be deployed every month.
+
+**Notes on acquisition:** N/A
+
+**Notes of caution:** Only `mobile` [methods](#deployment_type) can use this functionality.
+
+#### &lt;deployment_years&gt;
+
+**Data type:** List of integers
+
+**Default input:** N/A
+
+**Description:** A list of years used for scheduling. Methods can only be deployed during these years. For example, [2017,2018] indicates methods can only be deployed in 2017 and 2018\. If not defined, LDAR-Sim assumes methods can be deployed every year.
+
+**Notes on acquisition:** N/A
+
+**Notes of caution:** Only `mobile` [methods](#deployment_type) can use this functionality.
+
+### &lt;weather_envelopes&gt;
+
+**Description:** The following parameters define the valid weather conditions for the given method.
```

Comment:
Can we include information about what happens when weather is not inside valid conditions for a site on a given day?